### PR TITLE
Update advanced-hunting-urlclickevents-table.md

### LINKED ISSUE #11604: Support for GCC customers?
Is this table supported in GCC environments? If not, can this document be updated to reflect that please? Thank you

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0096bb8f-89d1-f968-4f1e-19e82b162ca5
* Version Independent ID: 0096bb8f-89d1-f968-4f1e-19e82b162ca5
* Content: [UrlClickEvents table in the advanced hunting schema](https://learn.microsoft.com/en-us/microsoft-365/security/defender/advanced-hunting-urlclickevents-table?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender/advanced-hunting-urlclickevents-table.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender/advanced-hunting-urlclickevents-table.md)
* Service: **microsoft-365-security**
* Sub-service: **m365d**
* GitHub Login: @schmurky
* Microsoft Alias: **maccruz**

### PR DESCRIPTION
**Pull request by msbemba**
Added information about supported tenants

reference from author https://learn.microsoft.com/en-us/microsoft-365/security/defender/supported-event-types?view=o365-worldwide#hunting-tables-support-status-in-event-streaming-api

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11604

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender/advanced-hunting-urlclickevents-table.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ ms.date: 04/27/2022
 - Microsoft 365 Defender
 
 
-The `UrlClickEvents` table in the advanced hunting schema contains information about [Safe Links](../office-365-security/safe-links-about.md) clicks from email messages, Microsoft Teams, and Office 365 apps in supported desktop, mobile, and web apps. 
+The `UrlClickEvents` table in the advanced hunting schema contains information about [Safe Links](../office-365-security/safe-links-about.md) clicks from email messages, Microsoft Teams, and Office 365 apps in supported desktop, mobile, and web apps.  UrlClickEvents is currently in public preview for Microsoft 365 Commercial customers and not available yet for GCC, GCC High or DoD.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/advanced-hunting-urlclickevents-table.md]:
```suggestion
The `UrlClickEvents` table in the advanced hunting schema contains information about [Safe Links](../office-365-security/safe-links-about.md) clicks from email messages, Microsoft Teams, and Office 365 apps in supported desktop, mobile, and web apps.  UrlClickEvents is currently in public preview for Microsoft 365 Commercial customers and not available yet for GCC, GCC High, or DoD.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@schmurky Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@schmurky Gentle reminder

**Pull request merged**